src/features/collapsible-sidebar/CollapsibleSidebar.scss

```diff
@@ -133,8 +139,34 @@ $left-sidebar-z-index: 40 !default;
     }
 }
 
-@include large-size {
-    .bdl-CollapsibleSidebar-logo {
-        margin-left: $collapsible-side-bar-h-padding + $bdl-grid-unit * 2 - 1px /* border */;
+@include breakpoint($medium-screen) {
+    .Body.is-responsive-web {
+        .bdl-CollapsibleSidebar-wrapper,
+        .bdl-CollapsibleSidebar {
+            width: 0;
+        }
+
+        .bdl-CollapsibleSidebar-footer,
+        .bdl-CollapsibleSidebar-nav ul {
+            display: none;
+        }
+
+        .bdl-CollapsibleSidebar-nav ul {
+            // using fixed width due to performance issues during expand transition
+            width: calc(100vw - (#{$collapsible-side-bar-h-padding} * 2));
+        }
+
+        .bdl-CollapsibleSidebar {
+            border: none;
```

Comment:
Does this mean that the sidebar doesn't have a border even when it is expanded?

Reply:
For mobile screens only, yeah. No need for a border at all on the sidebar when on mobile because it's either 0 width or 100% screen width.

Reply:
That is a great point.